model/model.js

```diff
@@ -89,7 +125,7 @@ class Admin{
         var sql = 'INSERT INTO leagueAdmin (leagueName, admin) VALUES ($1, $2) RETURNING *;';
         var admin = this.name
         console.log("Connecting " + leagueName + " to " + admin);
```

Comment:
connectLeague(leagueName){
        //This function connects a league to an administrator.
        var sql = 'INSERT INTO leagueAdmin (leagueName, admin) VALUES ($1, $2) RETURNING *;';
        var admin = this.name
        console.log("Connecting " + leagueName + " to " + admin);
        db.query(sql, [leagueName, admin]).then(result => {
            (result.rows[0])? console.log("Connected "+ leagueName + " to " + admin) : console.log("Did not connect "+ leagueName + " to" + admin)
            return
        }).catch(e => {
            console.log("\nLEAGUE CONNECTION ERROR!\n", e);
            return e;
        })
    }